src/components/liveblog/avatar.tsx

```diff
@@ -1,20 +1,18 @@
 // ----- Imports ----- //
 
-import React from 'react';
+import React, { ReactElement } from 'react';
 import { css, SerializedStyles } from '@emotion/core';
 
 import { Contributor, isSingleContributor } from 'contributor';
-import { src } from 'image';
-import { ImageMappings } from 'components/shared/page';
 
 
 // ----- Styles ----- //
 
-const imageWidth = 68;
+const imageWidth = '4rem';
```

Comment:
Any use of using the design system? Can't see a `remSpace[16]` though

Reply:
My understanding of the spacing system in Source was that it was designed for margins and paddings rather than widths, but I could be wrong 🤔